WordPress/Classes/Services/PeopleService.swift

```diff
@@ -148,6 +148,33 @@ struct PeopleService {
             failure(error)
         })
     }
+
+    /// Validates Invitation Recipients.
+    ///
+    /// - Parameters:
+    ///     - usernameOrEmail: Recipient that should be validated.
+    ///     - role: Role that would be granted to the recipient.
+    ///     - completion: Closure to be executed on completion. The boolean will indicate whether the OP
+    ///       was successful, or not.
+    ///
+    func validateInvitation(usernameOrEmail: String, role: Role, completion: (Bool -> Void)) {
+        remote.validateInvitation(siteID, usernameOrEmail: usernameOrEmail, role: role, completion: completion)
+    }
+
+
+    /// Sends an Invitation to a specified recipient, to access a Blog.
+    ///
+    /// - Parameters:
+    ///     - usernameOrEmail: Recipient that should be validated.
+    ///     - role: Role that would be granted to the recipient.
+    ///     - message: String that should be sent to the users.
+    ///     - completion: Closure to be executed on completion. The boolean will indicate whether the OP
+    ///       was successful, or not.
+    ///
+    func sendInvitation(usernameOrEmail: String, role: Role, message: String?, completion: (Bool -> Void)) {
```

Comment:
You could use a default value for the `message` parameter, and remove `unwrappedMessage`.

```swift
func sendInvitation(usernameOrEmail: String, role: Role, message: String = "", completion: (Bool -> Void)) {
```